addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -562,6 +574,10 @@ Object.assign(MessageList, {
         haveMessagesMarkAsReadIcon: Boolean,
         haveMessagesReplyIcon: Boolean,
         hasScrollAdjust: Boolean,
+        getScrollableElement: {
+            type: Function,
+            optional: true,
+        },
```

Comment:
Would provide docstring to this prop, explain what this is, why it's needed, etc.

This is especially handy for planned future work to clean all components. To give a quick taste of it: we want components to basically to be a UI reflection of models. If we follow this approach to the extreme, then components will have bare minimum props (and sometimes none). The way we deal scrollable part will very likely change drastically, so comment on this prop would help recall the complexity we had and increase changes to not do any regression while improving the code.